docs/zilevm/developer-rpc.md

```diff
@@ -0,0 +1,74 @@
+---
+id: developer-rpc
+title: RPC
+keywords:
+  - RPC
+description: Specification of ZILEVM RPC Endpoint
+---
+
+---
+
+## RPC API specification
```

Comment:
This could have a better title. We aren't giving a specification for an API. We're simply stating which bits of an already specified API we support.

What about "Supported JSON-RPC APIs"? @valeryz ?